smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSink.java

```diff
@@ -168,6 +180,48 @@ private void handleWriteResult(AsyncResult<?> ar, Message<?> message, ProducerRe
                 kafkaHeaders);
     }
 
+    private void createOutgoingTrace(Message<?> message, String topic, int partition, Headers headers) {
+        if (configuration.getTracingEnabled()) {
+            Optional<TracingMetadata> tracingMetadata = TracingMetadata.fromMessage(message);
+
+            final Span.Builder spanBuilder = TRACER.spanBuilder(topic + " send")
```

Comment:
is the "send" word defined somewhere? In Kafka, we use "write" most of the time.

Reply:
It's defined by OpenTelemetry specification. See https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/messaging.md#conventions

Reply:
great, don't forget to add this to the documentation.